backend/FwLite/LcmCrdt/FullTextSearch/EntrySearchService.cs

```diff
@@ -243,6 +253,9 @@ private static EntrySearchRecord ToEntrySearchRecord(Entry entry, WritingSystem[
 
     public IAsyncEnumerable<EntrySearchRecord> Search(string query)
```

Comment:
💭  If `Search` is only used by tests, would it be worth changing `public` to `internal` (to match this class's `EntrySearchRecords`, `internal` and also used only in tests)?

Reply:
I think `Search` is generally intended to be part of the public API. We just don't have a use for it yet.